src/lib/utils.js

```diff
@@ -488,6 +490,34 @@ const validateActorName = (actorName) => {
     }
 };
 
+const getPythonCommand = (directory) => {
+    const pythonVenvPath = /^win/.test(process.platform)
+        ? './.venv/bin/python3.exe'
```

Comment:
isn't it backslashes? or does the exec function automatically fix that?

Reply:
I think Node automatically fixes that on Windows. I'm installing Windows today, so I will test it.

https://nodejs.org/api/path.html#pathsep

> On Windows, both the forward slash (/) and backward slash (\\) are accepted as path segment separators; however, the path methods only add backward slashes (\\).